### PR TITLE
added attribution note as well as acknowledgments

### PR DESCRIPTION
**Pull request by 29td**
Added attribution note and acknowledgements.
Copied the previous code in order to review

### CODE REVIEW

**Review by 29td — COMMENTED** [review]
I think in your style.css you should decrease your styling (e.g header .logo and .fa-bars::before ).
they have the same styling if possible you can include them in one class.

**Review by 29td — COMMENTED** [review]
I see all linters are checked good job in adding all the linters. 💯

**Review by 29td — COMMENTED** [review]
Great job in using the correct html5 tags
👍

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+  padding: 15px 24px;
+  font-family: 'Arial, Helvetica', sans-serif;
+  justify-content: space-between;
+  display: flex;
+  color: rgb(255, 255, 255);
+  box-sizing: border-box;
+  width: 100%;
+  background: #3c3a39;
+}
+
```
Comment (29td) [inline css/style.css:25]: I think in your style.css you should decrease your styling (e.g header .logo and .fa-bars::before ).
they have the same styling if possible you can include them in one class.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+header .logo {
+  color: #fff;
+  text-decoration: none;
+}
+
+.fa-bars::before {
+  color: #fff;
+  text-decoration: none;
+}
```
Comment (29td) [inline css/style.css:34]: I think in your style.css you should decrease your styling (e.g header .logo and .fa-bars::before ).
they have the same styling if possible you can include them in one class.

**Comment by 29td** [comment]
> I think in your style.css you should decrease your styling (e.g header .logo and .fa-bars::before ). they have the same styling if possible you can include them in one class.



> I think in your style.css you should decrease your styling (e.g header .logo and .fa-bars::before ). they have the same styling if possible you can include them in one class.

File: css/style.css (hunk 2 above)
Comment (ishimwezachee) [reply css/style.css:34]: - [x] Your statement should not start with` I think`, you have to suggest a student some changes to his/her project and explain why you need those changes.

**Review by ishimwezachee — COMMENTED** [review]
Hi @29td , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️

- [x] Your project is should have at least two comments in the review. For now, you only have one comment, so you can add some more reviews to fulfill the requirements

- [ OPTIONAL]  I don't know why you've chosen this project, but if I am right you were supposed to use the project from Trials

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+  gap: 24px;
+}
+
+.work-item {
+  margin-top: 80px;
+}
+
+.work-item img {
+  display: block;
+
```
Comment (29td) [inline css/style.css:142]: Please remove the comment as it makes your code unprofessional.

**Review by 29td — COMMENTED** [review]
### Changes required
good job in making all linters checked 👍 

_Please check comments below in order to make required changes: _

File: css/style.css (hunk 2 above)
Comment (29td) [inline css/style.css:34]: You should decrease your styling (e.g header .logo and .fa-bars::before ).
they have the same styling if possible you can include them in one class.

**Review by 29td — COMMENTED** [review]
### Changes required
great use of html5 sematic tags 💯 

_please check below the required changes:_

**Review by ishimwezachee — APPROVED** [review]
Hi @29td , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**